_episodes_rmd/04-data-structures-part1.Rmd

```diff
@@ -63,12 +63,14 @@ cats
 ```
 
 
-The `read.csv` function is used for reading in tabular data stored in a text
-file where the columns of data are delimited by commas (csv = comma-separated
-values). Tabs are also commonly used to separated columns - if your data are in
-this format you can use the function `read.delim`. If the columns in your data
-are delimited by a character other than commas or tabs, you can use the more
-general and flexible `read.table` function.
+The `read.table` function is used for reading in tabular data stored in a text
+file where the columns of data are delimited by punctuation characters such as
+CSV files (csv = comma-separated values). Tabs and commas are the most common
+punctuation characters used to delimit data points in csv files. For
+convenience R provides 2 other versions of `read.table`. These are `read.csv`
+for files where the data is separated with commas and `read.delim` for files
+where the data is separated with tabs. If needed it is possible to override the
```

Comment:
I think in this case it would be clearer to use 'data' as plural, so 'data ia' becomes 'data are'.